src/main/java/fi/metatavu/metaform/server/rest/MetaformsApiImpl.java

```diff
@@ -422,6 +480,14 @@ public Response replyExport(UUID metaformId, UUID replyId, String format) {
     
     try {
       byte[] pdfData = replyController.getReplyPdf(metaform.getExportTheme().getName(), metaformEntity, replyEntity, attachmentMap, locale);
+      if (attachmentMap.isEmpty())
```

Comment:
Logging exported replies here is useless, the user has exported everything from the reply, so we know already that it contained all the attachments